app/views/users/show.html.erb

```diff
@@ -1,9 +1,32 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
-  <article class="timeline">
     <h3>Recent posts:</h3>
     <ul class="posts">
       <%= render @posts %>
     </ul>
-  </article>
+  <ol>
+  <% @user.friends.each do |friend| %>
+      <% if @user == current_user %>
+<li>
+ <%= friend.name %>
+        <%= button_to "Unfriend",
+                      friendships_destroy_path(friendship_id: Friendship.find_friendship(@user.id, friend.id)), class: "btn btn-secondary" %>
+      <% end %>
+    </li>
+  <% end %>
+</ol>
+<h3> These people wants to be friend with <strong> <%= @user.name %> </strong> </h3>
+<ol>
+  <% @user.inverse_friendships.each do |friendship| %>
+ <li>
+      <%= User.find(friendship.user_id).name %><br>
+      <%= button_to "Accept",
+                    friendships_update_path(friendship_id: friendship.id), class: "btn btn-secondary", onclick: "disable" %>
+      <%= button_to "Reject",
+                    friendships_destroy_path(friendship_id: friendship.id) %>
+    </li>
+</ol>
+ <% end %>
```

Comment:
- [ ] Friendly reminder, there should be absolutely no logic in views. Logic includes loop statements too. . Kindly make use of [helpers ](http://codefol.io/posts/Where-Do-I-Put-My-Code/)as they are  used to extract complex logic out of the view so that you can organize your code better